rules-tests/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector/Fixture/different_variable_name_used.php.inc

```diff
@@ -27,9 +27,9 @@ namespace Rector\Tests\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromo
 class DifferentVariableNameUsed
 {
     /**
-     * @param int $ID
+     * @param int $id
      */
-    public function __construct(private $ID)
+    public function __construct(private $id)
```

Comment:
@kick-the-bucket I just re-check and it seems `$id` should be kept as `$ID`, as property is named:

```php
private $ID;
```

this make property name changed from `$ID` to `$id`

Reply:
Oh, I see that it will change property usage, its ok then 👍